languages/nim/reference/exercise-concepts/protein-translation.md

```diff
@@ -0,0 +1,56 @@
+# Concepts of protein-translation
+
+[Example Implementation](https://github.com/exercism/nim/blob/master/exercises/protein-translation/example.nim)
+(Note: Excpetion raising ignored, as it isn't tested for)
```

Comment:
I'd include it in this extraction, as we are just extracting what is in the source code without regarding what is being tested for.

Reply:
Done!